### PR TITLE
[Docs] More docs on building the docs: pip and requirements-dev.txt [ci skip]

### PR DESCRIPTION
**Pull request by kripken**

### CODE REVIEW

**Comment by kripken** [comment]
ping @sbc100

File: docs/process.md [hunk 1]
```diff
@@ -103,6 +103,9 @@ that is:
 3. Run `make install EMSCRIPTEN_SITE=\[path-to-a-checkout-of-the-site-repo\]`
 3. Go to the site repo, commit the changes, and push.
 
+You will need the proper sphinx version installed, which you can do using
```
Comment (sbc100) [inline docs/process.md:106]: proper sounds a little judgmental maybe?   Maybe "specific"?

**Review by sbc100 — APPROVED** [review]

File: docs/process.md (hunk 1 above)
Comment (kripken) [reply docs/process.md:106]: Sure

Comment (kripken) [inline docs/process.md:106]:
```suggestion
You will need the specific sphinx version installed, which you can do using
```

**Pull request merged**